float64 joint_commanded_effort
float64 joint_measured_effort
string joint_name
float64 joint_position
float64 joint_velocity
